website/docs/install/install-with-bazel.md

```diff
@@ -35,7 +35,7 @@ These hardware specifications are recommended, but not required to run the Prysm
 ## Dependencies
 
 * A modern UNIX operating system
-* [Bazel](https://docs.bazel.build/versions/3.7.0/install.html) version **3.7.0** installed (Ensure you are using 3.7.0)
+* [Bazelisk](https://docs.bazel.build/versions/main/install.html) this will automatically manage the version of ***Bazel*** required. 
```

Comment:
```suggestion
* [Bazelisk](https://docs.bazel.build/versions/main/install-bazelisk.html) this will automatically manage the version of ***Bazel*** required. 
```